astropy/cosmology/io/mapping.py

```diff
@@ -0,0 +1,183 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+"""
+The following are private functions, included here **FOR REFERENCE ONLY** since
+the io registry cannot be displayed. These functions are registered into
+:meth:`~astropy.cosmology.Cosmology.to_format` and
+:meth:`~astropy.cosmology.Cosmology.from_format` and should only be accessed
+via these methods.
+"""
+
+import copy
+from collections.abc import Mapping
+
+import numpy as np
+
+from astropy.cosmology.core import _COSMOLOGY_CLASSES, Cosmology
+from astropy.io import registry as io_registry
+from astropy.table import QTable
+
+__all__ = ["from_mapping", "to_mapping"]
+
+
+def from_mapping(map, *, move_to_meta=False, cosmology=None):
```

Comment:
In previous conversation with @mhvk it was suggested to make the module ``_io`` -> ``io`` since my example code for writing a custom ``read/write/to_format/from_format`` included ``to/from_mapping``. I've since updated everything to exclusively use the publicly scoped ``Cosmology.to_format("mapping")``. Rather than making these functions prefixed with an underscore, I think I'll revert the module rename and make it explicitly private.